addons/survey/controllers/main.py

```diff
@@ -152,23 +158,26 @@ def _redirect_with_error(self, access_data, error_key):
     # ------------------------------------------------------------
     # TEST / RETRY SURVEY ROUTES
     # ------------------------------------------------------------
-
-    @http.route('/survey/test/<string:survey_token>', type='http', auth='user', website=True)
-    def survey_test(self, survey_token, **kwargs):
+    @http.route(['/survey/test/<int:survey_id>/<string:survey_token>',
+                '/survey/test/<string:survey_token>'
+                ], type='http', auth='user', website=True)
```

Comment:
:thinking: 'tis a bit ugly, does it lint properly?

Would add a comma after the last route and go to next line after the first `[`